src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --textColorPrimary: #060b35;
+  --textColorSecondary: #616070;
+  --buttonCyan: #00acdc;
+  --lhSmall: 14px;
+  --lhBig: 18px;
+  --fsSmall: 10px;
+  --fsBig: 12px;
+  --fwLow: 400;
+  --fwMid: 500;
+  --fwHigh: 700;
+}
```

Comment:
It's duplicated (exists in card.css too). This place is good for this, but you can also put it in a seperate file which would be a better approach

Reply:
I will try to do that, I couldn't use variables from other files, that's why that was there... I will try to put it in another file

Reply:
For some reason moving it to a separate file seems to have broken the stars... but on the bright side, using the variables from just inside style.css works fine now